client/client_test.go

```diff
@@ -495,7 +487,7 @@ func (s *ClientSuite) TestLocalExpired(c *C) {
 
 	// locally expired targets.json is ok
 	version = client.targetsVer
-	c.Assert(s.repo.AddTargetWithExpires("foo.txt", nil, s.expiredTime), IsNil)
+	c.Assert(s.repo.AddTargetsWithExpires(nil, s.expiredTime, "foo.txt"), IsNil)
```

Comment:
I agree and was hoping for input on this. Doing it this way allows us to use the `...` notation for paths and transparently handle one or many paths with the same function. If you have a look at client.go I was able to remove the singular versions of the Add/Remove functions because the pluralized versions now transparently handle one or many paths. The trade off is nillable fields at the front.